### DISCUSSION TITLE
article/let-nature-take-its-course

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#58] [Announcements]
# article/let-nature-take-its-course

💡 《道德经》被誉为万经之王，是中国历史上最伟大的名著之一，对中国哲学、科学、政治、宗教等产生了深刻影响。
据联合国教科文组织统计，《道德经》是除了《圣经》以外被译成外国文字发布量最多的文化名著。 《道德经》其要在乎理身、理国。

https://tangly1024.com/article/let-nature-take-its-course

**Comment by happyzhangyyds** [comment]
很有趣的理解和分享
